deploy/helm/tests/__snapshot__/node-collector-config-map_test.yaml.snap

```diff
@@ -165,19 +2584,9 @@ Node collector config should match snapshot when autodiscovery is disabled:
             metric:
             - type == METRIC_DATA_TYPE_HISTOGRAM
         filter/logs:
-          logs:
-            include:
-              match_type: regexp
-              resource_attributes:
-              - key: k8s.namespace.name
-                value: ^kube-.*$
-        filter/metrics:
-          metrics:
-            exclude:
-              match_type: regexp
-              metric_names:
-              - .*_temp
-              - apiserver_request_total
+          logs: |-
```

Comment:
What does `|-` do there? Are you sure that this works?

Reply:
No. I forgot a leftover debug code in there and it caused this. Fixed.